web/src/pages/InvoiceList/InvoiceList.js

```diff
@@ -154,77 +237,35 @@ function InvoiceList() {
-                expandIcon={<ArrowRight />}
-                aria-controls="panel1d-content"
-                id="panel1d-header"
-              >
-                <Typography variant="h6">Filters</Typography>
-              </AccordionSummary>
-              <AccordionDetails>
-                {/* Autocompletewidget grid container start */}
-                <Grid Container spacing={3} style={{ display: 'flex', alignItems: 'center' }}>
-                  <Grid item xs={6} sm={2}>
-                    <AutocompleteWidget options={countries} label="Country" disablePortal autoSelect size="small" />
-                  </Grid>
-                  <Grid item xs={6} sm={2}>
-                    <AutocompleteWidget options={countries} label="Office" disablePortal autoSelect size="small" />
-                  </Grid>
-                  <Grid item xs={6} sm={2}>
-                    <AutocompleteWidget options={countries} label="Salesmen" disablePortal autoSelect size="small" />
-                  </Grid>
-                  <Grid item xs={6} sm={2}>
-                    <AutocompleteWidget options={countries} label="Servicemen" disablePortal autoSelect size="small" />
-                  </Grid>
-                  <Grid item xs={6} sm={2}>
-                    <TextField fullWidth label="Customer" size="small" />
-                  </Grid>
-                  <Grid item xs={6} sm={2}>
-                    <TextField fullWidth label="Proj.Inv.OnAc.No" size="small" />
-                  </Grid>
-                  <Grid item xs={6} sm={2}>
-                    <AutocompleteWidget options={countries} label="Status" disablePortal autoSelect size="small" />
-                  </Grid>
-                  <Grid item xs={6} sm={2}>
-                    <BasicDatePicker label="Date" inputFormat="dd-MM-yyyy" views={['year', 'month', 'day']} />
-                  </Grid>
-                  <Grid item xs={6} sm={2}>
-                    <BasicDatePicker label="Date" inputFormat="dd-MM-yyyy" views={['year', 'month', 'day']} />
-                  </Grid>
-                  <Grid item xs={6} sm={2}>
-                    <Button variant="contained" size="small">
-                      Filter
-                    </Button>
-                  </Grid>
-                </Grid>
-                {/* Autocompletewidget grid container end */}
-              </AccordionDetails>
-            </Accordion>
-          </Grid>
+        <Grid item xs={12}>
+          <Filters
+            components={FILTER_COMPONETS}
+            apiUrl="dummyUrl"
+            getFilterData={getFilterData}
+            getFilterDataPayloadChange={getFilterDataPayloadChange}
+          />
         </Grid>
-        {/* Filter section xs={12} end */}
       </Grid>
       {/* Grid for simple table */}
       <Grid container spacing={3} style={{ marginTop: '1rem' }}>
         <Grid item xs={12}>
           <SimpleTable
-            rowData={InvoiceListingData}
+            rowData={InvoiceData}
```

Comment:
should use camelcase consistency => invoiceData

Reply:
Updated